docs/architecture/decisions/index.md

```diff
@@ -4,3 +4,4 @@
 - [2. Registration & Accreditation form database schema](0002-registration-accreditation-form-database-schema.md)
 - [3. GOV.UK Notify](0003-gov-uk-notify.md)
 - [4. Data Extraction](0004-data-extraction.md)
+* [5. Admin UI](0005-admin-ui-adr.md)
```

Comment:
The fight between `*` and `-` is causing PR checks to fail on formatting...

Reply:
Should be fixed now